CppCoreGuidelines.md

```diff
@@ -3959,27 +3959,45 @@ Another reason is been to delay initialization until an object is needed; the so
 * (Simple) Every constructor should initialize every member variable (either explicitly, via a delegating ctor call or via default construction).
 * (Unknown) If a constructor has an `Ensures` contract, try to see if it holds as a postcondition.
 
-### <a name="Rc-default0"></a> C.43: Give a class a default constructor
+### <a name="Rc-default0"></a> C.43: Ensure a class a default constructor
```

Comment:
The change from "Give" to "Ensure" is good, but it now makes this sentence ungrammatical.